apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/commons/QueryUtils.java

```diff
@@ -17,6 +19,47 @@
 
 /** As set of methods to help form JPA queries. */
 public class QueryUtils {
+  /**
+   * Database function that checks all claims for a given beneficiaryId and returns a bitwise mask
+   * value that shows if a given claim type will have any data.
+   */
+  public static final String CHECK_CLAIMS_FOR_DATA_SQL =
+      "SELECT * FROM check_claims_mask(:beneIdValue)";
+
+  /** BitSet index identifier for Carrier Claims. */
+  public static final int CARRIER_HAS_DATA = 0;
+  /** BitSet index identifier for Inpatient Claims. */
+  public static final int INPATIENT_HAS_DATA = 1;
+  /** BitSet index identifier for Outpatient Claims. */
+  public static final int OUTPATIENT_HAS_DATA = 2;
+  /** BitSet index identifier for SNF Claims. */
+  public static final int SNF_HAS_DATA = 3;
+  /** BitSet index identifier for DME Claims. */
+  public static final int DME_HAS_DATA = 4;
+  /** BitSet index identifier for HHA Claims. */
+  public static final int HHA_HAS_DATA = 5;
+  /** BitSet index identifier for Hospice Claims. */
+  public static final int HOSPICE_HAS_DATA = 6;
+  /** BitSet index identifier for Part D Events. */
+  public static final int PART_D_HAS_DATA = 7;
+
+  /** bitwise value denoting CARRIER_CLAIMS data for a beneficiary. */
+  public static final int V_CARRIER_HAS_DATA = (1 << 0);
```

Comment:
Can we use the values defined above in these declarations? E.g.
`public static final int V_CARRIER_HAS_DATA = (1 << CARRIER_HAS_DATA);`

Reply:
I suppose you could, but that ONLY works for CARRIER_CLAIMS since that has bit 1, so a value of 1. All the others go way beyond their index usage value.

Reply:
+1 to Billie's suggestion. I don't see why it wouldn't work and it would make the code clearer.

Reply:
How do you mean? Isn't it a 1:1 translation? Like if `OUTPATIENT_HAS_DATA = 2`, and `V_OUTPATIENT_HAS_DATA = (1 << 2);`, aren't you just replacing the hard-coded `2` with the variable `OUTPATIENT_HAS_DATA` which is also `2`? And then doing that for all of them? AKA

`V_SNF_HAS_DATA =  (1 << SNF_HAS_DATA);`
`V_DME_HAS_DATA = (1 << DME_HAS_DATA);`

it would be one less hard-coded value in my mind

Reply:
I don't disagree with either you your thoughts on this; am not going to do it right now as there is a follow-up ticket that will run the EOBs in parallel and I intend to make this bit handling encapsulated in an EnumSet for that. Also, changing the static values to be something like:
rslt.set(CARRIER_HAS_DATA, (maskVal & (1 << CARRIER_HAS_DATA)) != 0);
results in having to replicate that shifting logic in a lot of places which introduces more changes than warranted for code that will change to an EnumSet.

Reply:
ahh, I see what you were saying; thx, done.